api/lib/infrastructure/serializers/csv/csv-serializer.js

```diff
@@ -36,6 +36,10 @@ function deserializeForSessionsImport({ parsedCsvData, hasBillingMode }) {
   const csvPrepaymentCodeKey = headers.prepaymentCode;
   const firstCsvLine = parsedCsvData[0];
 
+  if (!firstCsvLine) {
```

Comment:
Ca pinaille, mais pour etre explicite: 
```suggestion
function isHeaderMissing(firstCsvLine) { return !firstCsvLine}
if (isHeaderMissing(firstCsvLine)) {
```

Reply:
ça ne vérifie pas la présence des headers, juste qu'il y a un élément dans le tableau des parsedCsvData

  const firstCsvLine = parsedCsvData[0];

Reply:
Tout à fait, tu as raison